src/App.test.js

```diff
@@ -1,12 +1,28 @@
+require('regenerator-runtime/runtime');
+
 const React = require('react');
 const Testing = require('@testing-library/react');
-const MiddleEnd = require('strange-middle-end');
-const Config = require('./middle-end/config');
-const App = require('./App');
+
+const createSandbox = () => {
+
+    const sandbox = {};
+
+    jest.isolateModules(() => {
```

Comment:
Any test that involves `./middle-end`, will need this chunk of code, correct?  If so, should we pull `createSandbox` out into a util of sorts?